src/libs/ReportActionsUtils.js

```diff
@@ -105,11 +105,16 @@ function isWhisperAction(action) {
 }
 
 /**
+ *
+ * Returns whether the comment is a thread parent message/the first message in a thread
+ *
  * @param {Object} reportAction
+ * @param {String} reportID
  * @returns {Boolean}
  */
-function hasCommentThread(reportAction) {
-    return lodashGet(reportAction, 'childType', '') === CONST.REPORT.TYPE.CHAT && lodashGet(reportAction, 'childVisibleActionCount', 0) > 0;
+function isThreadParentMessage(reportAction, reportID) {
+    const {childType, childVisibleActionCount = 0, childReportID} = reportAction;
```

Comment:
Just noting here: Previously we used `lodashGet` to possibly handle null / undefined `reportAction` - now that we're destructuring directly from `reportAction` this could cause errors if `reportAction` isn't defined, right? Is it possible `reportAction` isn't defined when this is called?

Reply:
Nice, I like this solution 👍  Thanks @fedirjh & @Nodebrute for implementing 👍